projects/aca-content/src/lib/components/header/header.component.html

```diff
@@ -1,19 +0,0 @@
-<adf-layout-header
-  [logo]="logo$ | async"
-  [redirectUrl]="landingPage"
-  [tooltip]="appName$ | async"
-  [color]="headerColor$ | async"
```

Comment:
this requires cleaning all the documentation related to these properties you remove